### PR TITLE
Ymdk_np21 via port

### PR DESCRIPTION
**Pull request by Zereef**
## Description

Ymdk_np21 via port

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/ymdk_np21/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Zereef 2021
```
Comment (drashna) [inline keyboards/ymdk_np21/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by Zereef** [comment]
added the GPL2+ compatible license header

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by Zereef** [comment]
this is already tested and being used on a daily basis

**Comment by drashna** [comment]
Thanks!

**Pull request merged**